app/assets/javascripts/discourse/app/components/site-header.js

```diff
@@ -38,17 +36,34 @@ const SiteHeaderComponent = MountWidget.extend(
     },
 
     _animateOpening($panel) {
-      $panel.css({ right: "", left: "" });
+      const $headerCloak = $(".header-cloak");
```

Comment:
Maybe it's worth removing all the jQuery while you're in there?

Reply:
sure, can do!